src/libs/import.c

```diff
@@ -710,6 +710,17 @@ static guint _import_set_file_list(const gchar *folder, const int folder_lgth,
 
   GFileInfo *info = NULL;
   guint nb = n;
+  /* get filmroll id for current directory. if not present, checking the db whether
```

Comment:
That's better! I would also put this routine into film.h/c

`dt_film_get_id(const char *folder);`

Reply:
Indeed.